docs/rules.md

```diff
@@ -49,6 +49,13 @@ hide:
     11. Do not write or install software on computers belonging to other players or servers without their informed consent.
         1.  When advertising malicious software, it should be made very clear that it is malicious during every interaction.
     12. Do not use No Chat Reports' chat encryption feature in public chat. Using it in private messages is allowed however, as long as the receiving party is able to decode it.
+    13. Any form of in game gambling must be fair to the players.
+        1. Gambling must *never* involve money that has value in the real world (FIAT, Crypto, etc.) directly or indirectly.
+        2. Any odds must be clearly displayed on an ingame monitor, poster or sign and immediately updated when changed, and in the user interface where the player gambles. In the user interface where the player gambles, only the applicable odds may be shown to decrease clutter, but it is recommended to show all odds.
+        3. When a gambling operator chooses to not use Kromer, regular items or Krist directly in the gambling machines, and chooses to use a coin system with an exchange rate, the following rules are applicable.
+            1. The exchange rate for the coins to ingame currency may be changed, but within reason and non frequent.
+            2. Any currency gained while gambling must be able to be paid out by the gambling operator to the player. For example, when a player wins a prize and the player earns 100 coins, those 100 coins should be able to be paid out at all times to the player.
+            3. If there isn't an exchange rate, this should be properly communicated to players on an ingame monitor, poster or sign and in the user interface where the player gambles.
```

Comment:
This is not organized, this rule only applies on the above clause 3. But directly contradicts it